ansible/app_options_files/public_testnet

```diff
@@ -0,0 +1,132 @@
+    "accounts": [
```

Comment:
So far I didn't see any problems with it. This contains public wallet addresses. No private keys are stored. People can check what kind of coins we have in our network, but as soon as bucky and the others start spending it, it's going to be a different amount.